README.md

```diff
@@ -67,3 +67,57 @@ teracy-dev:
         branch: develop
       require_version: ">= 0.4.0-SNAPSHOT"
 ```
+
+## Supported configuration
+These are the supported configuration in teracy-dev-core: `variables`, `vagrant`, `default`, `node`.
+You can see the details about the default configuration in the `config.yaml` file.
+
+
+### Examples
+
+This section gives you some examples about the supported configuration in teracy-dev-core, so you should know how
+to override the default configuration.
+
+You should add your new configuration into the `workspace/teracy-dev-entry/config_override.yaml`.
+
+*vagrant*
+
```

Comment:
Specify what this config does and what it contains

Reply:
It is mentioned in the config.yaml, why do we need to mention it here anymore?